docs/categories.yml

```diff
@@ -123,3 +123,5 @@ site:
   - Video
   - Web Development
   - WordPress
+  - Ghost
+  
```

Comment:
Remove line break, cuz that is not in the original file

```suggestion
  - Ghost
```

Reply:
Hi, 

Thanks for the help. I've removed line breaks. Let me know.

-Tanmay